packages/@angular/cli/utilities/dynamic-path-parser.ts

```diff
@@ -3,7 +3,8 @@ import * as process from 'process';
 import * as fs from 'fs';
 const stringUtils = require('ember-cli-string-utils');
 
-export function dynamicPathParser(project: any, entityName: string, appConfig: any) {
+export function dynamicPathParser(project: any, entityName: string, appConfig: any,
+                                  dryRun: boolean) {
```

Comment:
I think the options change would be a nice one, for when additional things are added. It will also simplify the defaulting of values assuming a class is used instead of an interface.